resources/dicts/patrols/plains/border/newleaf.json

```
@@ -1 +1,3695 @@
+                }
+            },
+            {
+                "text": "This burrow is weird. app1's never seen a tunnel with straight walls, or metal ones, for that matter. But c_n can always benefit from knowing about new tunnels, so {PRONOUN/app1/subject} {VERB/app1/creep/creeps} in. The trap door slams shut behind {PRONOUN/app1/object}.",
+                "exp": 0,
+                "weight": 20,
+                "art": "pln_hunt_gonetwolegtrap1",
+                "lost_cats": ["app1"]
+            },
+            {
+                "text": "This is going to be such a cool story to tell back at camp, app1 tells {PRONOUN/app1/self}. Checking the tunnel out, it looks like a collpased roof into a wider tunnel system, smelling faintly of something app1 can't identify. Huh, app1 wonders what-",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_train_patrolapprenticesessionsuccess",
+                "stat_trait": [
+                    "adventurous", 
+                    "rebellious", 
+                    "daring",
+                    "bold",
+                    "childish",
+                    "playful"
+                ],
+                "dead_cats":["app1"],
+                "history_text": {
+                    "scar": "m_c was injured after disturbing a wrathful prairie dog.",
+                    "reg_death": "This cat was killed as an apprentice, patrolling the border alone and investigating things {PRONOUN/m_c/subject} shouldn't have.",
+                    "lead_death": "patrolling the border alone and investigating things {PRONOUN/m_c/subject} shouldn't have"
+                }
+            },
+            {
+                "text": "Okay, so that might be a tunnel. app1 should go check it out, right? c_n cares about tunnels and stuff. {PRONOUN/app1/subject/CAP} clamber in, and then there's a sharp pain.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_dead_cat_app_EXPLICIT",
+                "art_clean": "gen_dead_cat_app",
+                "stat_trait": [
+                    "oblivious", 
+                    "grumpy"
+                ],
+                "dead_cats":["app1"],
+                "history_text": {
+                    "scar":"",
+                    "reg_death": "This cat was killed as an apprentice, patrolling the border alone and investigating things {PRONOUN/m_c/subject} shouldn't have.",
+                    "lead_death": "patrolling the border alone and investigating things {PRONOUN/m_c/subject} shouldn't have"
+                }
+            },
+            {
+                "text": "The shallow cave is home to a young badger, inexperienced, dangerous, and far from its natal family. Larchpaw escapes, battered, bruised, and bleeding, but the subadult predator won't follow Larchpaw back to camp. Not in territory that's unfamiliar and new to it.",
```

```suggestion
                "text": "The shallow cave is home to a young badger, inexperienced, dangerous, and far from its natal family. app1 escapes, battered, bruised, and bleeding, but the subadult predator won't follow app1 back to camp. Not in territory that's unfamiliar and new to it.",
```

WHO is larchpaw lmao